test/cross-browser-testing/babel.config.js

```diff
@@ -0,0 +1,12 @@
+export default {
```

Comment:
Nitpick: Can we name this `babel.CBT.config.js` so that it is consistent name with the rollup config in the same file?